index1.html

```diff
@@ -0,0 +1,104 @@
+              <li class="content-liC"><a href="#">CC compaign</a></li>
+         
+          </div>
+
+          <div class="header-desk">
+              <div class="header-first">
+                     <li><a href="#" target="_blank"><i class="fab fa-facebook"></i></a></li>
+                     <li> <a href="https://twitter.com/zoumaniguimass1" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a></li>
+                     <ul class=" nav-headerfirst">
+                            <li><a href="#">English</a></li>
+                            <li><a href="#">My page</a></li>
+                            <li><a href="#">loyout</a></li>
+                     </ul>
+              </div>
+              <div class="header-second">
+               <div class="logo"><a href="index.html"><img src="images/icon.png " width="100%" height="80%" alt=""></a></div>
+               <nav>
+                      <ul>
+                             <li class="content-liA"><a href="index1.html">About</a></li>
+                             <li class="content-liA"><a href="#">program</a></li>
+                             <li class="content-liA"><a href="#">Join</a></li>
+                             <li class="content-liA"><a href="#">Sponsor</a></li>
+                             <li class="content-liP"><a href="#">News</a></li>
+                             <li class="content-liCc"><a href="#">CC compaign</a></li>
+                      </ul>
+               </nav>
+        </div>
+              </div>
+          </div>
+ </header>
+
+ <div class="welcom">
+  <h1 class="welcom-h1">Developed by young developers for the improvement of national education</h1>
+
+  <p class="welcom-p1">
+      
+  <p class="welcom-p2">
+   The integration of Artificial Intelligence (AI) in education assessment is a cutting-edge technological advancement that is revolutionizing the traditional way of correcting exams. With the use of AI, the process of grading exams becomes faster, more accurate, and less subjective. This technology uses algorithms that are programmed to recognize patterns and identify correct answers based on a set of criteria.
+  </p>
+  <p class="welcom-p3">
+     <span>Please contact us for more information and investment  <a href="#">zoumaniguimass12@gmail.com</a></span>
+   
+  </p>
+
+ </div>
+
+<div class="welcom1">
+
+   <div class="welcom11" >Topic goal</div>
+   <div class="welcom12" ></div>
+   <div class="welcom13" >Overall, furfur represents an exciting and rapidly-evolving field within AI in education, with the potential to revolutionize the way that students learn and teachers teach. As the technology continues to advance and improve, it is likely that we will see furfur and other AI-based systems play an increasingly important role in education at all levels.
+   </div>
+   <div class="welcom14" ><img class="welcom14img" src="images/fontAI1.jpg"  width="100%" height="100%" alt=""> </div>
+</div>
```

Comment:
- [ ] Kindly follow the design guideline to match the design.
  - Every content should be `centered` for both mobile and desktop versions.
  - For `welom14` class kindly use `logo` instead of any other images.

- Given Design:
![image](https://user-images.githubusercontent.com/60083437/227038889-c1205a4a-4292-4c24-8c24-55f85da4239e.png)